pkg/workload/kv/kv.go

```diff
@@ -121,6 +125,13 @@ func (w *kv) Hooks() workload.Hooks {
 			}
 			return nil
 		},
+		PostLoad: func(sqlDB *gosql.DB) error {
+			if w.secondaryIndex {
+				_, err := sqlDB.Exec(`CREATE INDEX IF NOT EXISTS v_idx ON kv (v)`)
```

Comment:
if you keep this as PostLoad, it should be marked as a runtime-only flag, since it doesn't affect fixtures here